Bizyhood/Shortcodes.php

```diff
@@ -0,0 +1,330 @@
+<?php
+
+// Make sure we don't expose any info if called directly
+if ( !function_exists( 'add_action' ) ) {
+	echo 'Hi there!  I\'m just a plugin, not much I can do when called directly.';
+	exit;
+}
+
+/**
+ * This class contains all Bizyhood Shortcodes.
+ */
+ 
+class Bizyhood_Shortcodes
```

Comment:
Again, all classes have the prefix, before we even start customizing the forked code.